[Feat]: print helpful snippet after running generate ABI script

haroune-mohammedi opened the pull request.

haroune-mohammedi committed:
feat: print helpful snippet after running generate ABI script

ivpavici reviewed (commented):
I've tested this and looks good :)
I would just add this snippet to README
`starkli class-at "0x00000005dd3d2f4429af886cd1a3b08289dbcea99a294197e9eb43b0e0325b4b" --network mainnet | npx abi-wan-kanabi --input /dev/stdin --output abi.ts
`

And maybe note that this was tested with Starkli version 0.2.2

haroune-mohammedi committed:
doc: add a section about usage with starknet.js

The section also mentions the fact that we can get started with
one command combining both `starkli` and `npx abi-wan-kanabi`
